Cargo.toml

```diff
@@ -361,7 +361,7 @@ zstd = { version = "0.11.2", default-features = false }
 leveldb-sys = { git = "https://github.com/vectordotdev/leveldb-sys.git", branch = "leveldb_mmap_limit" }
 # Removes dependency on `time` v0.1
 # https://github.com/chronotope/chrono/pull/578
-chrono = { git = "https://github.com/vectordotdev/chrono.git", branch = "no-default-time-v0.4.22" }
+chrono = { git = "https://github.com/vectordotdev/chrono.git", branch = "no-default-time-v0.4.22-1" }
```

Comment:
@bruceg just curious what this was for?

Reply:
The chrono `Item` type was missing an implementation of `Hash`, which was needed for the upstream users of `Template`. Implementing it locally would need a newtype wrapper, which would have been a PITA, so I implemented it upstream.

That reminds me, I got the changes merged into `chrono` master, but the team would like a backport for the v0.4 branch as well.